### ISSUE TITLE
Partnering with IsDev Conference

### ISSUE THREAD

**Issue by OpesanyaAdebayo** [nodejsnigeria/ecosystem#12]
A co-organizer for IsDev conference told me about it sometime last week. I thought Node.js Nigeria could be a part of the conference for the following reasons:
- It usually involves a hackathon, so various technologies will be used. It will be nice to see us represented there.
- It gives us visibility and helps the rest of the ecosystem know that we're an active community.
- In partnering with other dev communities, we are a step closer in achieving our goal to build a diverse, well-rounded ecosystem.

Seeing that we cannot financially support at the moment, I proposed to them that we could do the following:
- Make a blog post about our partnership and why we encourage our community to be a part
- Send an email to our database (which is over 1,000 people)
- Promotion on Twitter

In exchange for a small logo on the conference website and letting us bring a standing banner to the conference.
I'm waiting on the organizers for feedback. I'll post it here once they get back to me.
Please let me know if you have ideas or concerns to share.

[Here's a blog post of the 2017 edition of the conference](https://medium.com/isdev-nigeria/what-happened-isdev2017-hackathon-7bc79736a062)

**Comment by pmbanugo** [comment]
Any link to their website?

**Comment by OpesanyaAdebayo** [comment]
Closing this issue because the conference has been postponed till further notice.
